### DISCUSSION TITLE
PIR Sensor works till first power disconnection

### DISCUSSION THREAD

**Discussion by szaman-89** [arendst/Tasmota#14630] [Q&A] [answered]
Hello,
I have a problem with fully converting LSC Flood Light with Sensor (light with PIR  and daylight sensor from Action) to latest tasmota firmware with ESP-12F.
Problem is in PIR sensor functionality. After configuring all things in firmware everything works as expected. Problem occurs when device is unpluged and pluged again to power source. After that PIR sensor stops working and nothing what I know can't make it work again. Only reseting firmware and configuring everything again solve problem. Of course another power down mess with PIR functionality again. I'm out of ideas...

**Comment by ascillato2** [comment, marked as answer]
Hi, please, can you share some logs when it works and when it stops working?
Also, please, share the gpio configuration and the rules you use

**Reply by szaman-89** [reply]
Hello, problem was with soldering all available pads on lsc's board to esp. Desoldering one of gpio's fixed problem.

**Comment by dkliszewski** [comment]
How did you manage to disassemble the flood light lamp?

**Reply by szaman-89** [reply]
You need to disassembly pir case only. Case is welded so you need to use penknife to open it. Be careful to not cut board itself.

**Reply by sfromis** [reply]
Better cut your fingers! :grinning:

**Reply by dkliszewski** [reply]
THX - I founded :)
